requirements.txt

```diff
@@ -1,4 +1,3 @@
 click
 requests
 pyyaml
-statusdb
```

Comment:
hej @robinandeer, I believe `statusdb` should be in this list though it is not in `PyPi` (yet). It is cause we have [this](https://github.com/SciLifeLab/TACA/blob/master/dependency_links.txt) file and for this to work we should also mention it in `requirement.txt` :)